test/test_pebble.py

```diff
@@ -27,8 +27,10 @@
 
 import pytest
 import websocket  # type: ignore
+from testing import Harness
```

Comment:
We're testing `ops/testing.py` functionality here, so this should go in `test_testing.py`, not `test_pebble.py`. This test should live in or alongside the existing [`test_get_pebble_container_plan`](https://github.com/canonical/operator/blob/dc47640f8ea915049d9e20a1fe4ea189c96ae8d1/test/test_testing.py#L2760) test there.